web/pdf_thumbnail_viewer.js

```diff
@@ -13,25 +13,9 @@
  * limitations under the License.
  */
 
-'use strict';
-
-(function (root, factory) {
-  if (typeof define === 'function' && define.amd) {
-    define('pdfjs-web/pdf_thumbnail_viewer', ['exports',
-      'pdfjs-web/ui_utils', 'pdfjs-web/pdf_thumbnail_view'], factory);
-  } else if (typeof exports !== 'undefined') {
-    factory(exports, require('./ui_utils.js'),
-      require('./pdf_thumbnail_view.js'));
-  } else {
-    factory((root.pdfjsWebPDFThumbnailViewer = {}), root.pdfjsWebUIUtils,
-      root.pdfjsWebPDFThumbnailView);
-  }
-}(this, function (exports, uiUtils, pdfThumbnailView) {
-
-var watchScroll = uiUtils.watchScroll;
-var getVisibleElements = uiUtils.getVisibleElements;
-var scrollIntoView = uiUtils.scrollIntoView;
-var PDFThumbnailView = pdfThumbnailView.PDFThumbnailView;
+import { getVisibleElements, scrollIntoView, watchScroll
+  } from 'pdfjs-web/ui_utils';
```

Comment:
I think this could be improved in terms of how we write multiline imports. Personally I would find the following easier to read:

```javascript
import {
  getVisibleElements, scrollIntoView, watchScroll
} from 'pdfjs-web/ui_utils';
```

This looks more like the notation we use for objects. I don't know if there are actual conventions for this, but otherwise we should choose something that is the most readable and use it consistently.